gpii/node_modules/contextManager/src/ContextManager.js

```diff
@@ -169,6 +169,7 @@
 
         lifecycleManager.update(finalPayload, function (response) {
             if (response.success) {
+                delete finalPayload.lifecycleInstructions;
```

Comment:
I'd add a comment here summarising some of this information - it's hard enough for the reader of the code to capture where they are in such a complex lifecycle. Should explain the last sentence of your comment in particular